Water Drink Water/src/clients/blazor.wa.tbd/Pages/Log.razor

```diff
@@ -12,4 +12,54 @@
             <button class="btn btn-primary" onclick="@(async () => await LogConsumption())">Log</button>
         </div>
     </div>
+    @if (false) { // "feature flag"
+        <div class="row">
+            <div class="col-md-12">
+                <input id="log-date-picker" 
+                       type="date" 
+                       min="2020-01-01"
+                       max="2030-01-01"
```

Comment:
Where do these min / max values come from? Did we agree on these somewhere?

Reply:
Think we discussed this on stream, but we did not agree on these values.